install/kubernetes/helm/istio/charts/sidecar-injector/templates/istio-sidecar-injector.yaml

```diff
@@ -0,0 +1,93 @@
+{{- $proxy_image := dict "image" "" }}
+{{- if .Values.global.proxy.debug }}
+{{- $_ := set $proxy_image "image" "proxy_debug" }}
+{{- else }}
+{{- $_ := set $proxy_image "image" "proxy" }}
+{{- end }}
+{{- $istio_namespace := .Values.global.namespace }}
+{{- $pilot_hub := .Values.global.pilot.hub }}
+{{- $pilot_tag := .Values.global.pilot.tag }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: istio-sidecar-injector
+  namespace: {{ $istio_namespace }}
+  labels:
+    istio: sidecar-injector
+spec:
+  ports:
+  - port: 443
+  selector:
+    istio: sidecar-injector
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: istio-sidecar-injector
+  namespace: {{ $istio_namespace }}
+  labels:
+    istio: sidecar-injector
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: istio-sidecar-injector
+      labels:
+        istio: sidecar-injector
+    spec:
+      serviceAccountName: istio-sidecar-injector-service-account
+      containers:
+        - name: webhook
+          image: {{$pilot_hub}}/sidecar_injector:{{$pilot_tag}}
+          imagePullPolicy: IfNotPresent
+          args:
+            - --tlsCertFile=/etc/istio/certs/cert.pem
```

Comment:
Is this the istio-CA provisioned cert, or the webhook cert ? If it is not the istio-CA - it would create a lot of confusion, please use different dir. ( can be fixed in a separate PR)

Reply:
k8s-CA provisioned cert. Have a recommendation for alternative directory?